src/reference/faq.rst

```diff
@@ -70,24 +71,56 @@ The other deciding factor is if you're unable to install Docker (e.g. for admins
 The ambient runtime is not available without WSL.
 
 
+.. old anchors
 .. _why-intel-miniconda-installer-on-apple-silicon:
 
-Why recommend the Intel Miniconda installer for Mac computers with Apple silicon (e.g. M1)?
--------------------------------------------------------------------------------------------
+.. _why-conda-install-errors-on-apple-silicon:
+
+Why do I get an error when installing to my Conda environment on a Mac computer with Apple silicon (e.g. M1)?
+-------------------------------------------------------------------------------------------------------------
+
+An example error:
+
+.. code-block:: text
+   :emphasize-lines: 9,10,11,12,13
+
+   (your-environment-name) $ mamba install augur
+
+   Looking for: ['augur']
+
+   bioconda/osx-arm64
+
+   …
+
+   Could not solve for environment specs
+   Encountered problems while solving:
+     - nothing provides mafft needed by augur-10.0.0-py_0
+
+   The environment can't be solved, aborting the operation
+
+This happens when using an ARM64-based Conda installation on a `computer with Apple silicon <https://support.apple.com/en-us/HT211814>`__, but there are workarounds.
+
+Apple silicon chips are great and efficient. They are based on a different chip architecture, ARM64/AArch64, and come with performance improvements compared to the x64-based Intel chips in older Macs.
+
+However, many existing packages have not yet added support to run on these chips natively. An easy way to identify support on the `Bioconda packages page <https://anaconda.org/bioconda>`_ is to look for ``noarch`` or ``osx-arm64`` under the **Installers** section of a package. Without any of those, a package is not able to be installed natively on Apple silicon. This is the case for packages such as `MAFFT <https://anaconda.org/bioconda/mafft>`_ (a dependency of :term:`Augur`) and many other bioinformatics packages. For this reason, using an ARM64-based Conda installation for the average bioinformatics researcher can result in a difficult experience.
+
+There are two ways to work around this:
+
+1. Uninstall Conda, delete all existing environments, and re-install with an `Intel-based installer <https://docs.conda.io/en/latest/miniconda.html>`__. With an Intel-based installation, all environments are forced to use emulation.
 
-.. note::
+   This provides easy compatibility with a broader set of bioinformatics packages, but comes at the cost of relatively longer run times for packages that have native ARM64 support.
 
-   `How to tell if your Mac has an Apple silicon chip <https://support.apple.com/en-us/HT211814>`_
+2. Create a custom Conda environment that installs and runs packages under `Intel emulation <https://conda-forge.org/docs/user/tipsandtricks.html#installing-apple-intel-packages-on-apple-silicon>`__. Run this after setting up an **empty** Conda environment and before installing any packages to it:
```

Comment:
This link instructs users to specify the subdir in two steps:
```
CONDA_SUBDIR=osx-64 conda create -n ...
...
conda config --env --set subdir osx-64
```
but in the next line you imply that only the second is necessary. (I don't actually know the answer here.)

Also, can you clarify if the `conda config --env --set subdir osx-64` step ever needs to be run again (for that environment), or whether once is enough?

Reply:
The first line is useful if you plan to create and install packages in the same command, e.g.

```sh
# This will create an environment and install osx-arm64 Python.
conda create -n your-environment-name python

# This will create an environment and install osx-64 Python.
CONDA_SUBDIR=osx-64 conda create -n your-environment-name python
```

But that is only an env var that doesn't apply to future `conda install` commands. You'd still have to set the `subdir` config regardless.

My suggestion here (which I now realize deviates from the linked example) is to create an **empty** environment and set the `subdir` config before installing anything.

Reply:
> Also, can you clarify if the `conda config --env --set subdir osx-64` step ever needs to be run again (for that environment), or whether once is enough?

Clarified in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/427b38c5be7d9649469743731f01a57d3ed9a4b4..4366c1b002d7f78291ce281cb478fa45b53ed886):

> You only need to run this once per Conda environment.

Reply:
Thanks! I find apple silicon causes me headaches rather regularly because of these things. I keep meaning to write them up (I _cannot_ install `jax`, for instance, no matter what I try), but these things are hard to write up. 

> My suggestion here (which I now realize deviates from the linked example) is to create an empty environment and set the subdir config before installing anything.

Yeah, that's the approach I use too. Probably because you told me to! It works well for me until something needs to compile, then all bets are off.

Reply:
The confusion is real and I hope to continue improving the installation / FAQ pages to clear things up, at least for Nextstrain-related usage.

Separately, I'm curious about your difficulties installing `jax`. Taking that to [Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1683914627972929).